x/wasm/IBC.md

```diff
@@ -0,0 +1,137 @@
+# IBC specification
```

Comment:
This file also should be rewritten. so I think it would be better to remove it for now.

Reply:
Ok, removed it.